app/networkapi/settings.py

```diff
@@ -83,7 +83,6 @@
     'adminsortable',
 
     'networkapi.people',
-    'networkapi.features',
```

Comment:
inconvenient as it sounds, let's not overload PRs: remove features in a different PR